public/style/global.scss

```diff
@@ -171,6 +171,17 @@ c-content {
   background-color: #e8e8e8;
 }
 
+iframe.stackblitz{
```

Comment:
We don't really need stackblitz iframe css since we use codepen

Maybe for later